app/src/pages/create.tsx

```diff
@@ -217,7 +217,7 @@ function Create() {
           required
         />
         {/* TODO: add tinymce integration */}
-        <Form.Input name="description" displayName="Description" type="text" />
+        <Form.TextArea name="description" displayName="Description" maxLength={200} />
```

Comment:
nit: i would define a constant in the `formUtils.ts` file and export it. this way it is defined in one place and can be reused.

```typescript
export const MAX_DESCRIPTION_LENGTH = 200;
```

```suggestion
        <Form.TextArea name="description" displayName="Description" maxLength={MAX_DESCRIPTION_LENGTH} />
```